mpc-core/src/ot/extension/kos15/receiver/state.rs

```diff
@@ -0,0 +1,36 @@
+use super::BaseSender;
+use super::KosMatrix;
+use crate::Block;
+use rand_chacha::ChaCha12Rng;
+
+pub trait ReceiverState {}
+
+pub struct Initialized {
+    pub base_sender: BaseSender,
+    pub rng: ChaCha12Rng,
+    pub cointoss_share: [u8; 32],
```

Comment:
What is the benefit of this? We then have to write a lot of getters/setters because `Kos15Sender` and `Kos15Receiver` do not have access anymore, since state is in its own module.

Reply:
:+1: